zp-relayer/workers/sentTxWorker.ts

```diff
@@ -203,6 +204,19 @@ export async function createSentTxWorker<T extends EstimationType>(gasPrice: Gas
           await job.update({
             ...job.data,
           })
+        } else if (isInsufficientBalanceError(err)) {
+          // We don't want to take into account last gasPrice increase
+          job.data.prevAttempts.pop()
```

Comment:
I think it's better to keep the tx hash regardless of error, the same way we treat gas price errors

Reply:
Agree, but I think we should not increase gas price in this case. Maybe just changing last gas price entry in the `prevAttempts` to previous one will be ok?

Reply:
Sure, should be fine